docs/Available commands and exit codes.md

```diff
@@ -0,0 +1,24 @@
+# vagrant-service-manager commands and exit codes
+
+Commands                                                | Explanations
+--------------------------------------------------------|-----------------------------------------
```

Comment:
Why putting this essential information into a sub page instead of the main README? There are other things which can be split out from there, if one wants to trim down the README.